### PR TITLE
(#19) [New]: Captcha added

### LINKED ISSUE #19: Add captcha
Add captcha in register forms.

Proposal: https://pypi.org/project/django-recaptcha/
We'll need a google's account.

### PR DESCRIPTION
**Pull request by francescarpi**
## Description

Django recaptcha added in register and new job form.

fixes #19 

## Code formating

The code has to be well formatted following the formatting rules. For example in Python the code has to follow the PEP8.

Before create your PR, please make sure your code is well formatted and styled doing:
```bash
$ >> pre-commit run --all
```

### CODE REVIEW

**Comment by francescarpi** [comment]
Hi @XaviTorello  @jbagot .

I already have done this PR but I don't know ho manage the private/public captcha keys.

We need to create one from here:

https://www.google.com/recaptcha/about/

But...

* ¿We need a dev credencials for put in `.dev` file?
* Is not necessary to do anything and you will put the keys in the kubernete's secrets?

Thank you!

File: jobs/settings.py [hunk 1]
```diff
@@ -314,3 +316,8 @@
 EMAIL_PORT = env("EMAIL_PORT", default=None)
 EMAIL_HOST_USER = env("EMAIL_HOST_USER", default=None)
 EMAIL_HOST_PASSWORD = env("EMAIL_HOST_PASSWORD", default=None)
+
+# Catpcha
+RECAPTCHA_PUBLIC_KEY = env("RECAPTCHA_PUBLIC_KEY", default="")
+RECAPTCHA_PRIVATE_KEY = env("RECAPTCHA_PRIVATE_KEY", default="")
```
Comment (jbagot) [inline jobs/settings.py:322]: The private key shouldn't have default. Then you have to add it in docker-compose.testing.yaml like the rest of secrets that are there.
After that I'll add it in the helm chart

**Review by jbagot — CHANGES_REQUESTED** [review]

**Comment by francescarpi** [comment]
> @francescarpi I left out the captcha from #149 waiting for this PR. May you add captcha on form page? Thanks

What form? Contact?

File: jobs/settings.py (hunk 1 above)
Comment (francescarpi) [reply jobs/settings.py]: Done! Thank you!

**Comment by francescarpi** [comment]
> > @francescarpi I left out the captcha from #149 waiting for this PR. May you add captcha on form page? Thanks
> 
> What form? Contact?

Done!

**Comment by francescarpi** [comment]
Waiting #151

**Comment by francescarpi** [comment]
Rebased with master and conflicts resolved.

File: jobsapp/forms.py [hunk 2]
```diff
@@ -44,6 +47,7 @@ class ContactForm(forms.Form):
     )
     subject = forms.CharField(max_length=256, help_text=_("Reason why you are contact us"))
     message = forms.CharField(max_length=2048, widget=forms.Textarea, help_text=_("Your message"))
+    captcha = ReCaptchaField(widget=ReCaptchaV2Checkbox)
```
Comment (fullonic) [inline jobsapp/forms.py:50]: We don't need captcha in our form. We are using one from formspree.io.

**Review by fullonic — COMMENTED** [review]
For me looks good. Just need to remove the captcha from the django form and it's it. Good job :+1:

**Pull request merged**